crates/sui-types/src/dynamic_field.rs

```diff
@@ -41,6 +41,23 @@ pub struct Field<N, V> {
     pub value: V,
 }
 
+/// Rust version of the Move sui::dynamic_object_field::Wrapper type
+#[derive(Clone, Copy, Serialize, Deserialize, Debug)]
+pub struct DOFWrapper<N> {
+    pub name: N,
+}
+
+impl<N> MoveTypeTagTrait for DOFWrapper<N>
+where
+    N: MoveTypeTagTrait,
```

Comment:
Not relevant to this PR, but I find the name of this trait and its function to be a bit strange in Rust